collect_app/src/main/java/org/odk/collect/android/activities/CustomTabHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.odk.collect.android.activities;
+
+import android.app.Activity;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsClient;
+import android.support.customtabs.CustomTabsService;
+import android.support.customtabs.CustomTabsServiceConnection;
+import android.support.customtabs.CustomTabsSession;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by sanjeev on 17/3/17.
+ */
+
+public class CustomTabHelper {
```

Comment:
Here also when user doesnt have custom chrome tabs support it is launched in browser.
Certainly but I think those ones are very few cases and It will enhance the experience of majority of users. Whats say?